src/p2p/tx.ts

```diff
@@ -1,6 +1,8 @@
-import { BufferReader } from './binaryReader';
-import { getEnumDescription } from './helpers';
-import { StacksMessageParsingError, NotImplementedError } from './errors';
+import { BufferReader } from '../binary-reader';
+import { getEnumDescription } from '../helpers';
+import { StacksMessageParsingError, NotImplementedError } from '../errors';
+import * as clarityUtil from '../../node_modules/stacks-transactions-js/src/clarity/clarityTypes';
+import * as stacksTxUtil from '../../node_modules/stacks-transactions-js/src/utils';
```

Comment:
Seems to import/build fine as `stacks-transactions-js/src/clarity/clarityTypes` when I tried? Is there a certain scenario this is needed?

Reply:
Thanks! This is a left over from debugging problems with git-importing this lib. Fixed now